shell/platform/android/BUILD.gn

```diff
@@ -481,119 +481,8 @@ action("pom_embedding") {
-    "test/io/flutter/external/FlutterLaunchTests.java",
-    "test/io/flutter/plugin/common/BinaryCodecTest.java",
-    "test/io/flutter/plugin/common/StandardMessageCodecTest.java",
-    "test/io/flutter/plugin/common/StandardMethodCodecTest.java",
-    "test/io/flutter/plugin/editing/InputConnectionAdaptorTest.java",
-    "test/io/flutter/plugin/editing/ListenableEditingStateTest.java",
-    "test/io/flutter/plugin/editing/TextInputPluginTest.java",
-    "test/io/flutter/plugin/localization/LocalizationPluginTest.java",
-    "test/io/flutter/plugin/mouse/MouseCursorPluginTest.java",
-    "test/io/flutter/plugin/platform/PlatformPluginTest.java",
-    "test/io/flutter/plugin/platform/PlatformViewsControllerTest.java",
-    "test/io/flutter/plugin/platform/SingleViewPresentationTest.java",
-    "test/io/flutter/plugins/GeneratedPluginRegistrant.java",
-    "test/io/flutter/util/FakeKeyEvent.java",
-    "test/io/flutter/util/PreconditionsTest.java",
-    "test/io/flutter/util/ViewUtilsTest.java",
-    "test/io/flutter/view/AccessibilityBridgeTest.java",
-  ]
-
-  outputs = [
-    depfile,
-    jar_path,
-    jar_path + ".md5.stamp",
-  ]
-
-  additional_jar_files = [ "robolectric.properties" ]
-
-  _jar_dependencies = [
-                        android_sdk_jar,
-                        embedding_jar_path,
-                      ] + embedding_dependencies_jars
-
-  inputs = _jar_dependencies + additional_jar_files
-
-  _rebased_current_path = rebase_path(".")
-  _rebased_jar_path = rebase_path(jar_path, root_build_dir)
-  _rebased_depfile = rebase_path(depfile, root_build_dir)
-  _rebased_classpath = rebase_path(_jar_dependencies, root_build_dir)
-  _rebased_srcjars = rebase_path(_jar_dependencies, root_build_dir)
-  _rebased_additional_files = rebase_path(additional_jar_files, root_build_dir)
-
-  if (host_os == "mac") {
-    _javacbin =
-        rebase_path("//third_party/java/openjdk/Contents/Home/bin/javac")
-    _jarbin = rebase_path("//third_party/java/openjdk/Contents/Home/bin/jar")
-  } else if (host_os == "win") {
-    _javacbin = rebase_path("//third_party/java/openjdk/bin/javac.exe")
-    _jarbin = rebase_path("//third_party/java/openjdk/bin/jar.exe")
-  } else {
-    _javacbin = rebase_path("//third_party/java/openjdk/bin/javac")
-    _jarbin = rebase_path("//third_party/java/openjdk/bin/jar")
-  }
-
-  args = [
-    "--depfile=$_rebased_depfile",
-    "--jar-path=$_rebased_jar_path",
-    "--classpath=$_rebased_classpath",
-    "--jar-source-base-dir=$_rebased_current_path",
-    "--java-srcjars=$_rebased_srcjars",
-    "--additional-jar-files=$_rebased_additional_files",
-    "--jar-bin=$_jarbin",
-    "--javac-bin=$_javacbin",
-  ]
-
-  args += rebase_path(sources, root_build_dir)
-
-  deps = [
-    ":android_jar",
-    ":flutter_shell_java",
-  ]
+# TODO(jsimmons): remove this placeholder when it is no longer used by the LUCI recipes
+group("robolectric_tests") {
```

Comment:
This rule is useful to make`robolectric_tests` depend on `android_jar`. Do you know if gn has something equivalent to `bazel test :<target>` or `bazel run :<target>`?

Reply:
I don't know of a specialized command like that.  But I suppose you could create a GN target with an action that runs `run_tests.py --type=java` and a dependency on the Android engine JAR.